scripts/cibw-cuda-setup.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
```

Comment:
```suggestion

set -eu
```

`-e` to exit if any command fails, `-u` to error on undefined bash variables